travel/admin.py

```diff
@@ -1,8 +1,31 @@
 from django.contrib import admin
+from registration.models import RegistrationProfile
+from registration.users import UsernameField
+from django.utils.translation import ugettext_lazy as _
+from .models import Event, Principal, Trip, Region, EventType, Agency, Office
 
-# Register your models here.
+class RegistrationReduxAdmin(admin.ModelAdmin):
+    actions = ['activate_users']
+    list_display = ('user','activated')
+    raw_id_fields = ['user']
+    search_fields = ('user__{0}'.format(UsernameField()),
+                     'user__first_name', 'user__last_name')
+    
+    def activate_users(self, request, queryset):
+        """
+        Activates the selected users, if they are not already
+        activated.
+        """
+        for profile in queryset:
```

Comment:
This only activates selected users and then deletes the registration profile for those users.  I just tested it to make sure as well.